src/tests/ftest/util/general_utils.py

```diff
@@ -601,17 +601,23 @@ def stop_processes(hosts, pattern, verbose=True, timeout=60):
     result = {}
     log = getLogger()
     log.info("Killing any processes on %s that match: %s", hosts, pattern)
+
+    # Filter out zombie (defunct) processes from ps output
+    ps_with_zombie_filter = "ps x | grep -E {} | grep -vE '\<(grep|defunct)\>'"
```

Comment:
(lint) '\<'. String constant might be missing an r prefix.
(lint) '\>'. String constant might be missing an r prefix.